README.md

```diff
@@ -28,7 +28,14 @@ cp config.example.php config.php
 
 Edit `config.php` with your Twilio credentials.
 
-Next, we'll need to install the Twilio Helper library via [Composer](https://getcomposer.org/).
+Next, we'll need to install the Twilio Helper library. Follow the instructions to install [Composer](https://getcomposer.org/).
+
```

Comment:
I don't think we should add OS specific install instructions here. Maybe we should say.

> If `composer install` fails ensure that you have added it correctly to your path by following the instructions on [Composer](https://getcomposer.org).